CMakeLists.txt

```
@@ -33,12 +40,14 @@ if (EDGE_SDK_NO_EXCEPTION)
     set(CMAKE_C_FLAGS "-fno-exceptions ${CMAKE_C_FLAGS}")
 endif()
 
+if (MSVC AND EDGE_SDK_PARALLEL_BUILD_ENABLE)
+    add_compile_options(/MP)
+    message(STATUS "Using parallel compiling (/MP)")
+endif ()
```

I saw you used `endif(MSVC)` in previous if-clause. Shouldn't it we use it here as well?
```suggestion
endif (MSVC AND EDGE_SDK_PARALLEL_BUILD_ENABLE)
```

---

Done